rpc/documentation/api.md

```diff
@@ -792,6 +838,137 @@ wallet and republished later if it or a double spend are not mined.
 
 ___
 
+#### `TicketPrice`
+
+The `TicketPrice` method returns the price of a ticket for the next block, also 
+known as the stake difficulty. May be incorrect if the daemon is currently 
+syncing.
```

Comment:
It is always syncing...

To avoid the issue where the returned ticket price is not correct for the next block, I would include either the current block height+hash or next block height in the response.

I also feel that there should be a notification system for receiving these updates.  Polling this sucks.